CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 
 ### Bug Fixes
 
+- Pass the validate function to the http manager. See [Issue #5242](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5242)
+- Change the resource id param for IMDS. See [Issue #5238](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5238)
+
+4.71.0
```

Comment:
Seems like there's two entries for 4.71.0 now?

Reply:
Good catch @Avery-Dunn! Thanks